style.css

```diff
@@ -1 +1,126 @@
-/*  Add your styling here */
+.grid {
+    /* border: solid 1px red; */
+
+}
+
+.grid-one {
+    display: grid;
+    grid-template-columns: 140px 140px;
+    /* border: solid 1px pink; */
+}
+
+.grid-one>img {
+    margin: auto;
+}
+
+.grid-one>h1 {
+    margin: auto;
+}
+
+@media screen and (min-width:900px) {
+    .grid-one {
+        display: flex;
+    }
+
+    .grid-one>img {
+        margin-left: 20px;
+    }
+
+    .grid-one>h1 {
+        margin-right: 100px;
+    }
+
+    .grid-two {
+        display: flex;
+        justify-content: space-around;
+    }
+
+    .general-hero-section {
+        display: flex;
+        padding: 10px;
+
+    }
+
+    .first-section {
+        width: 40%;
+        margin-right: 20px;
+    }
+
+    .second-section {
+        width: 60%;
+    }
+
+    .pictures-section {
+        display: grid;
+        grid-template-columns: auto auto auto auto;
+    }
+
+    .each-picture {
+        margin: 5px;
+    }
+
+    .footer {
+        justify-content: flex-start;
+    }
+}
+
+.grid-two {
```

Comment:
Watch out with class names like this - it implies that it does something to the layout but in reality it doesn't.